ui/desktop/app/routes/application.js

```diff
@@ -29,7 +29,10 @@ export default class ApplicationRoute extends Route {
    * reported by the main process.  If they differ, update the main process
    * clusterUrl so that the renderer's CSP can be rewritten to allow requests.
    */
-  @notifyError(({ message }) => message, { catch: true })
+  @notifyError(({ message }) => message, {
+    catch: true,
+    log: { origin: 'ApplicationRoute:beforeModel' },
```

Comment:
is this the route format we should follow when we log from a route's hook? I'm ok with this approach, I was just curious bc the above controller log has the action name

Reply:
I think this is a solid approach. Since it is a `beforeModel` hook, it isn't clear where it is coming from so I figured adding the class route name is a good approach.